tests/cephfs/clients/mds_conf_modifying.py

```diff
@@ -180,6 +181,11 @@ def run(ceph_cluster, **kw):
             )
             if output[0].rstrip() != target[1]:
                 raise CommandFailed(f"Failed to set {client_conf} to {value}")
+
+        rhbuild = config.get("rhbuild")
+        if "7" in rhbuild:
+            set_balance_automate_and_validate(client1)
```

Comment:
Hi Hemanth
Should we check the return value of the set_balance_automate_and_validate method and fail the test case if it fails any validation? Are we missing this step?

Reply:
Validation is being done in this function only Amar - 
def set_balance_automate_and_validate(client):

Should we have one more validation check in the main function?

Reply:
Updated Amar.